README.md

```diff
@@ -470,10 +471,39 @@ _Daniel Miao_
     - Associate build with version
     - Submit app for review
   - Add `Power and Performance Metrics and Diagnostics API`
-  
-
-
 
+## Accelerate your app with CarPlay
+
+https://developer.apple.com/wwdc20/10635 
+
+_Jonathan Hersh, Allen Langmaier_
+
+- **New CarPlay Templates***
```

Comment:
`**New CarPlay Templates***`VS `**New Templates**` is confusing to me, they sound like the same thing

Reply:
Should be fixed